packages/stv/src/index.test.ts

```diff
@@ -749,7 +784,7 @@ describe('redistributeToCandidates', () => {
         { totalVotes: 20, votes: [{ voteCount: 20, voteOrder: ['C', 'A'] }] },
       ],
     ]);
-    redistributeToCandidates(organizedVotes, candidateSet, 1, 10, 30);
+    redistributeToCandidates(organizedVotes, candidateSet, 1 / 3);
```

Comment:
Had to update the next few tests as less input is passed to this method. This was also part of #3, but I had to do it here as there was just no need for the excess parameters anymore.